centreon/src/Core/Common/Infrastructure/Repository/AbstractVaultRepository.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Core\Common\Infrastructure\Repository;
+
+use Centreon\Domain\Log\LoggerTrait;
+use Core\Security\Vault\Domain\Model\VaultConfiguration;
+use Core\Security\Vault\Application\Repository\ReadVaultConfigurationRepositoryInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class AbstractVaultRepository
```

Comment:
This class is not abstract ?
```suggestion
abstract class AbstractVaultRepository
```